### PR TITLE
Blog: adding tags, mainly for Dev portal and platform engineering

### PR DESCRIPTION
**Pull request by SaraDPH**
## Description
Multiple articles relate to developer portals & platform engineering, but they need to be tagged to be found together.

The following tags will be added for SEO purposes (since they become part of the page itself) and for easy findability via tags. 
    - developer-portals
    - platform_engineering

Update: I also added other missing tags  on other somewhat related articles 
    
## Checklist:

- [ ] I have reviewed the [style guide](https://github.com/pulumi/pulumi-hugo/blob/master/STYLE-GUIDE.md).
- [ ] If blogging, I have reviewed the [blogging guide](https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md).
- [ ] I have manually confirmed that all new links work.
- [ ] I added aliases (i.e., redirects) for all filename changes.
- [ ] If making css changes, I rebuilt the bundle.

### CODE REVIEW

**Comment by SaraDPH** [comment]
Note: I using the PR to also add Kat's video to her article

File: themes/default/content/blog/building-developer-portals/index.md [hunk 1]
```diff
@@ -37,6 +37,7 @@ tags:
 - templates
 - policy-as-code
 - backstage
+- platform_engineering
```
Comment (cnunciato) [inline themes/default/content/blog/building-developer-portals/index.md:40]:
```suggestion
- platform-engineering
```

File: themes/default/content/blog/developer-portal-gallery/index.md [hunk 2]
```diff
@@ -33,6 +33,9 @@ authors:
 # At least one tag is required. Lowercase, hyphen-delimited is recommended.
 tags:
     - features
+    - developer-portals
+    - platform_engineering
```
Comment (cnunciato) [inline themes/default/content/blog/developer-portal-gallery/index.md:37]:
```suggestion
    - platform-engineering
```

File: themes/default/content/blog/developer-portal-platform-teams/index.md [hunk 3]
```diff
@@ -16,6 +16,7 @@ tags:
     - templates
     - policy-as-code
     - backstage
+    - platform_engineering
```
Comment (cnunciato) [inline themes/default/content/blog/developer-portal-platform-teams/index.md:19]:
```suggestion
    - platform-engineering
```

File: themes/default/content/blog/environments-secrets-configurations-management/index.md [hunk 4]
```diff
@@ -14,6 +14,7 @@ authors:
 tags:
     - esc
     - secrets
+    - platform_engineering
```
Comment (cnunciato) [inline themes/default/content/blog/environments-secrets-configurations-management/index.md:17]:
```suggestion
    - platform-engineering
```

File: themes/default/content/blog/future-cloud-infrastructure-10-trends-shaping-2024-and-beyond/index.md [hunk 5]
```diff
@@ -36,7 +36,8 @@ tags:
     - iac
     - cloud-computing
     - multi-cloud
-    - forecast
+    - finops
+    - platform_engineering
```
Comment (cnunciato) [inline themes/default/content/blog/future-cloud-infrastructure-10-trends-shaping-2024-and-beyond/index.md:40]:
```suggestion
    - platform-engineering
```

File: themes/default/content/blog/remediation-policies/index.md [hunk 6]
```diff
@@ -1,10 +1,15 @@
 ---
 title: "Remediation Policies: Continuous and Automatic Compliance"
 authors: ["joe-duffy"]
-tags: ["pulumi-news"]
 meta_desc: "Pulumi CrossGuard remediation policies allow you to automatically fix violations, not just report them, ensuring continuous and automatic compliance."
 date: "2023-10-20"
 meta_image: "remediation.png"
+tags:
+    - features
+    - platform_engineering
```
Comment (cnunciato) [inline themes/default/content/blog/remediation-policies/index.md:9]:
```suggestion
    - platform-engineering
```

**Review by scottslowe — APPROVED** [review]

**Pull request merged**